neurodsp/sim/combined.py

```diff
@@ -82,3 +84,89 @@ def sim_combined(n_seconds, fs, components, component_variances=1):
     sig = np.sum(sig_components, axis=0)
 
     return sig
+
+
+@normalize
+def sim_central_freq(n_seconds, fs, central_freq, bw, ht, ap_func='sim_powerlaw',
+                     ap_kwargs={'exponent': -2}):
+    """
+    Returns a time series whose full power spectrum consists of an aperiodic component
+    and a gaussian peak at central_freq with standard deviation bw and relative height ht.
+
+
+    Parameters
+    -----------
+    n_seconds: float
+        Number of seconds elapsed in the time series.
+    fs: float
+        Sampling rate.
+    central_freq: float
+        Central frequency for the gaussian peak in Hz.
+    bw: float
+        Bandwidth, or standard deviation, of gaussian peak in Hz.
+    ht: float
```

Comment:
I don't find `ht` a super obvious label. Maybe we just spell it out as `height`.
I'm a little tempted to call it 'power' (maybe 'pow' or even 'pw'), but perhaps that isn't a technically ideal label?
Not totally sure on this, but _maybe_ we want to update this label.

Reply:
I have it as `height` for now as I'm working on addressing these comments. `pw` could also work since it matches fooof reports, as does `bw`.